runtime/rococo/src/lib.rs

```diff
@@ -855,19 +855,19 @@ impl pallet_parachain_staking::Config for Runtime {
 	type Currency = Balances;
 	type MonetaryGovernanceOrigin = EnsureRootOrAllCouncil;
 	/// Minimum round length is 2 minutes (10 * 12 second block times)
-	type MinBlocksPerRound = ConstU32<{ 2 * MINUTES }>;
+	type MinBlocksPerRound = ConstU32<{ prod_or_fast!(2 * MINUTES, 2) }>;
 	/// Blocks per round
-	type DefaultBlocksPerRound = ConstU32<{ 2 * MINUTES }>;
+	type DefaultBlocksPerRound = ConstU32<{ prod_or_fast!(2 * MINUTES, 2) }>;
```

Comment:
I am just curious here, Does changing the blocks per round affect the rewards being paid out?

Reply:
Potentially yes if `BlocksPerRound` setting is overriden on chain - but this flag will not be applied in prod anyway